### DISCUSSION TITLE
Windows defender threat warning

### DISCUSSION THREAD

**Discussion by MadOokami** [seerge/g-helper#569] [General]
Hello. I got this warning just now when trying to download version 0.80. This should be false positive no?
<img width="240" alt="image" src="https://github.com/seerge/g-helper/assets/82376847/55f982d8-23d1-47bf-9f62-f8150c59378c">

**Comment by seerge** [comment]
@MadOokami 
Hello, false positives from defender can happen. You can either download 0.79 or a 0.81 instead 
https://github.com/seerge/g-helper/releases/tag/v0.81
